src/glass/src/app/pages/services-page/services-page.component.ts

```diff
@@ -215,4 +228,29 @@ export class ServicesPageComponent {
       }
     });
   }
+
+  private removeService(serviceName: string): void {
+    this.dialogService.open(
+      DialogComponent,
+      (res: boolean) => {
+        if (res) {
+          this.blockUI.start(translate(TEXT('Please wait, deleting service ...')));
```

Comment:
If the action menu is `Remove service`, i suggest to use this wording in the waiting spinner too.
```suggestion
          this.blockUI.start(translate(TEXT('Please wait, removing service ...')));
```